_includes/mailchimp.html

```diff
@@ -50,6 +50,7 @@ <h2>Subscribe to our mailing list and join the Stakeholder Committee</h2>
         <div class="response" id="mce-success-response" style="display:none"></div>
     </div>    <!-- real people should not fill this in and expect good things - do not remove this or risk form bot signups-->
     <div style="position: absolute; left: -5000px;" aria-hidden="true"><input type="text" name="b_ecd1444846318d70d0cb9198a_96d745b70a" tabindex="-1" value=""></div>
+	<div><p><i>Note: Submitting this form will trigger a popup containing a captcha required to complete registration. You may need to set your browser to allow popups from our site.</i></p></div>
```

Comment:
let's say "may" instead of "will" since I don't think it always happens. Maybe say `an "I am not a robot" captcha`? not sure that everyone knows what a captcha is.